renderer/brave_content_renderer_client.cc

```diff
@@ -17,3 +22,13 @@ SetRuntimeFeaturesDefaultsBeforeBlinkInitialization() {
   blink::WebRuntimeFeatures::EnableSharedArrayBuffer(false);
 }
 BraveContentRendererClient::~BraveContentRendererClient() = default;
+
+#if defined(OS_ANDROID)
+void BraveContentRendererClient::RenderFrameCreated(
+	content::RenderFrame* render_frame) {
+  ChromeContentRendererClient::RenderFrameCreated(render_frame);
+
+  new cosmetic_filters_worker::CosmeticFiltersJsRenderFrameObserver(
+      render_frame, ISOLATED_WORLD_ID_CHROME_INTERNAL);
```

Comment:
we want maybe a new ID for this? Like the Greaselion one that is not used but that wrongly uses the same ID as Chrome's internal one?

Reply:
talked to Serg, the greaselion one was already removed and it's ok to share the same ID as chrome internal.